packages/synapse-react-client/src/components/DiscussionSearchResult.tsx

```diff
@@ -80,7 +80,7 @@ const DiscussionSearchResult = (props: DiscussionSearchResultProps) => {
   }
 
   return (
-    <div className="search-result-container">
+    <div className="bootstrap-4-backport search-result-container">
```

Comment:
Move `bootstrap-4-backport` to the component that needs it, rather than providing the class in a wrapping div
<img width="1181" alt="updated_ForumSearch" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/cb555458-f029-41e5-8b87-1421aa38566a">